kaolin/transforms/transforms.py

```diff
@@ -237,8 +264,13 @@ class RealignPointCloud(object):
 
     TODO: Example.
 
+    TODO: This transform's repr is incorrect:
```

Comment:
`tgt` is ignored in the `repr`, but it shouldn't be because it does change the output of the transformation. However I have yet to figure out a way to actually put it into the repr, since it could be a giant point tensor.

Reply:
It's hashed, only once per dataset constructor: https://github.com/NVIDIAGameWorks/kaolin/blob/master/kaolin/datasets/base.py#L90 I think you can just serialize it, you can also hash it.

Reply:
Done.